pyexchange/tethfinex.py

```diff
@@ -149,6 +149,8 @@ class TEthfinexToken(ERC20Token):
 
     abi = Contract._load_abi(__name__, 'abi/TETHFINEX.abi')
     bin = Contract._load_bin(__name__, 'abi/TETHFINEX.bin')
+    abi_token = Contract._load_abi(__name__, 'abi/TOKEN.abi')
```

Comment:
the new .abi files are missing from PR

Reply:
added `abi_locker` abi, deleted `abi_token` cause it is not used.